release/_bom.sh

```diff
@@ -114,28 +114,30 @@ function generate_api_source_jar {
 	done
 }
 
-function generate_pom_release_dxp_api {
-	local pom_file_name="release.dxp.api-${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}.pom"
+function generate_pom_release_api {
```

Comment:
@pgydev, where is the commit https://github.com/natocesarrego/liferay-docker/pull/51/commits/3f2e022ba0f21ed6147a6b4af44da02474844c6b?

Can you please include in this pull?